src/AutoRotatingCarousel.js

```diff
@@ -195,6 +195,7 @@ export default class AutoRotatingCarousel extends Component {
                 index={modulo(this.state.slideIndex, this.props.children.length)}
                 style={landscape ? style.dotsLandscape : style.dots}
                 onDotClick={this.handleChange}
+                dotColor={this.props.dotColor}
```

Comment:
This prop should be destructured above.

Reply:
Can you elaborate on what you mean by that? I'm not seeing any other destructuring of `this.props`.

Reply:
You're right… nevermind.

Reply:
:+1: